index.html

```diff
@@ -29,13 +32,144 @@ <h2 class="occupation">I’am a Software Developer</h2>
+                    <li><button class="btn boot">bootstrap</button></li>
+                    <li><button class="btn">Ruby</button></li>
+                </ul>
+                <button class="see pro">See Project</button>
+            </div>
+            <div class="grid-item">
+                <h3 class="professional-btn">Profesional Art Printing Data</h3>
+                <p class="multiple-p">A daily selection of privately <br> personalized reads; no accounts or
+                    <br>sign-ups required.
+                    has been the industry's standard</p>
+                <ul class="languages">
+                    <li><button class="btn">html</button></li>
+                    <li><button class="btn boot">bootstrap</button></li>
+                    <li><button class="btn">Ruby</button></li>
+                </ul>
+                <button class="see pro">See Project</button>
+            </div>
+            <div class="grid-item">
+                <h3 class="professional-btn">Profesional Art Printing Data</h3>
+                <p class="multiple-p">A daily selection of privately <br> personalized reads; no accounts or
+                    <br>sign-ups required.
+                    has been the industry's standard</p>
+                <ul class="languages">
+                    <li><button class="btn">html</button></li>
+                    <li><button class="btn boot">bootstrap</button></li>
+                    <li><button class="btn">Ruby</button></li>
+                </ul>
+                <button class="see pro">See Project</button>
+            </div>
+            <div class="grid-item">
+                <h3 class="professional-btn">Profesional Art Printing Data</h3>
+                <p class="multiple-p">A daily selection of privately <br> personalized reads; no accounts or
+                    <br>sign-ups required.
+                    has been the industry's standard</p>
+                <ul class="languages">
+                    <li><button class="btn">html</button></li>
+                    <li><button class="btn boot">bootstrap</button></li>
+                    <li><button class="btn">Ruby</button></li>
+                </ul>
+                <button class="see pro">See Project</button>
+            </div>
+            <div class="grid-item">
+                <h3 class="professional-btn">Profesional Art Printing Data</h3>
+                <p class="multiple-p">A daily selection of privately <br> personalized reads; no accounts or
+                    <br>sign-ups required.
+                    has been the industry's standard</p>
+                <ul class="languages">
+                    <li><button class="btn">html</button></li>
+                    <li><button class="btn boot">bootstrap</button></li>
+                    <li><button class="btn">Ruby</button></li>
+                </ul>
+                <button class="see pro">See Project</button>
+            </div>
```

Comment:
- [ ] Please,  Add the button styling (Hover, enable, Active, Pressed ). Kindly, use this style for all your buttons.
Figma design :  
![image](https://user-images.githubusercontent.com/71874425/153439044-5425e7da-e7cf-4629-924e-c975dc567c9e.png)